src/st/dai/dai.h

```diff
@@ -42,6 +42,13 @@
 #define NUM_CUTSCENE_PRIM 7
 #endif
 
+// These are used by both e_chair and e_confessional
+#define STAGE_HAS_CONFESSIONAL
+#define CONFESSIONAL_GHOST_PRIEST 0
+#define CONFESSIONAL_GHOST_PARISHIONER 1
+#define CONFESSIONAL_GHOST_BAD 0x100
+#define CONFESSIONAL_GHOST_READY 1
```

Comment:
I think typically we would put very specific defines like this in the c file (eg. DAI's `e_chair.c`) to keep the defines closer to the usage?

Eg. I am thinking https://github.com/Xeeynamo/sotn-decomp/blob/be8a585c9c6f69a051ec84cb833027d09052ed93/src/st/no1/e_collect.c#L4